src/site/content/en/blog/requestvideoframecallback-rvfc/index.md

```diff
@@ -0,0 +1,210 @@
+  // Re-register the callback to be notified about the next frame.
+  video.requestVideoFrameCallback(doSomethingWithTheFrame);
+};
+// Initially register the callback to be notified about the first frame.
+video.requestVideoFrameCallback(doSomethingWithTheFrame);
+```
+
+In the callback, `now` is a [`DOMHighResTimeStamp`](https://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp)
+and `metadata` is a [`VideoFrameMetadata`](https://wicg.github.io/video-rvfc/#dictdef-videoframemetadata)
+dictionary with the following properties:
+
+- `presentationTime`, of type `DOMHighResTimeStamp`:
+  The time at which the user agent submitted the frame for composition.
+- `expectedDisplayTime`, of type `DOMHighResTimeStamp`:
+  The time at which the user agent expects the frame to be visible.
+- `width`, of type `unsigned long`:
+  The width of the video frame, in media pixels.
+- `height`, of type `unsigned long`:
+  The height of the video frame, in media pixels.
+- `mediaTime`, of type `double`:
+  The media presentation timestamp (PTS) in seconds of the frame presented (e.g., its timestamp on the `video.currentTime` timeline).
+- `presentedFrames`, of type `unsigned long`:
+  A count of the number of frames submitted for composition. Allows clients to determine if frames were missed between `VideoFrameRequestCallback`s.
+- `processingDuration`, of type `double`:
+  The elapsed duration in seconds from submission of the encoded packet with the same presentation timestamp (PTS) as this frame (e.g., same as the `mediaTime`) to the decoder until the decoded frame was ready for presentation.
+
+For WebRTC applications, additional properties can appear:
+
+- `captureTime`, of type `DOMHighResTimeStamp`:
+  For video frames coming from either a local or remote source, this is the time at which the frame was captured by the camera.
+  For a remote source, the capture time is estimated using clock synchronization and RTCP sender reports
+  to convert RTP timestamps to capture time.
+- `receiveTime`, of type `DOMHighResTimeStamp`:
+  For video frames coming from a remote source, this is the time the encoded frame was received
+  by the platform, i.e., the time at which the last packet belonging to this frame was received over the network.
+- `rtpTimestamp`, of type `unsigned long`:
+  The RTP timestamp associated with this video frame.
+
+{% Aside %}
+  Note that `width` and `height` might differ from `videoWidth` and `videoHeight` in certain cases
+  (e.g., an anamorphic video might have rectangular pixels).
+{% endAside %}
+
+Of special interest in this list is ⏱ `mediaTime`.
+In Chromium's implementation, we use the audio clock as the time source that backs `video.currentTime`,
+whereas the `mediaTime` is directly populated by the `presentationTimestamp` of the frame.
+The `mediaTime` is what you should use if you want to exactly identify frames in a reproducible way,
+including to identify exactly which frames you missed.
+
+{%Aside 'warning' %}
```

Comment:
```suggestion
{% Aside %}
```

Reply:
There's no aside type because, as [the handbook says](https://web.dev/handbook/web-dev-components/#note-asides), this is supplemental information. Warnings and cautions are reserved for things requiring action from the reader.